### DISCUSSION TITLE
Anyone tried opi5 max?

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#990] [General]
How does it feel? Does wifi and bt work with jr ubuntu? Or armbian for that matter?
If i saw right, thers spi onboard so no issue booting of whatever? So a slight upgrade on many things with no cons. 
Thinking of puting original opi5 for some work and geting max to replace it for tinkering/deaktop/youtube/plex etc..

**Comment by Joshua-Riek** [comment]
I can not express how annoying the WiFi and Bluetooth drivers are for this board, but I got it working. But it's just another rk3588. Don't expect anything different besides IO.

**Comment by tyrius02** [comment]
I have four OPi 5 maxes. One seems to be defective, when I power it up with an SD card or eMMC it just goes into a boot / kernel panic loop.
The other three are working beautifully. I have a computer vision solution that uses the RKNPU and that appears to be working quite nicely, too.
